doc/source/dev/index.rst

```diff
@@ -173,6 +173,7 @@ Guidelines
 * No changes are ever committed without review and approval by a core
   team member.Please ask politely on the PR or on the `mailing list`_ if you
   get no response to your pull request within a week.
+* If you would like to create a new feature please get alignment first by discussing the feature in the `mailing list`_
```

Comment:
```suggestion
```
The addition of this line here might be a bit confusing, as these are general guidelines but the mailing list suggestion is only for new feature proposals. We don't want to make new users feel like they have to ping the mailing list before contributing anything.